sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/implementation/Utility.java

```diff
@@ -61,15 +62,15 @@ private Utility() {
             // current chunk don't have enough bytes so return true to get next Chunk if there is one.
             return true;
         })
-        .takeWhile(doContinue -> doContinue)
-        .then(Mono.defer(() -> {
-            if (contentType[0] != null) {
-                return Mono.just(contentType[0]);
-            } else {
-                return Mono.error(new RuntimeException("Content type could not be detected. "
-                    + "Should use other overload API that takes content type."));
-            }
-        }));
+            .takeWhile(doContinue -> doContinue)
```

Comment:
nitpick: seems like this spacing isn't as good as how it used to be

Reply:
Yeah, they were correctly formatted now.